simtools/Nodus.yaml

```diff
@@ -1,25 +1,7 @@
 - name: Nodus
 - website_url: ''
-- operating_system:
-  - windows: ''
-  - macOS: ''
-  - linux: ''
+- operating_system: ''
 - biological_level: ''
```

Comment:
This seems like a simulator "transportation freight network". Do we still include it?

Reply:
Lol, no I don't think we should. Wonder how it ended up on the compneuroweb model tools list…